pkg/integration/tests/commit/commit_skip_hooks.go

```diff
@@ -0,0 +1,35 @@
+package commit
+
+import (
+	"github.com/jesseduffield/lazygit/pkg/config"
+	. "github.com/jesseduffield/lazygit/pkg/integration/components"
+)
+
+var CommitSkipHooks = NewIntegrationTest(NewIntegrationTestArgs{
+	Description:  "Commit with skip hook using CommitChangesWithoutHook",
+	ExtraCmdArgs: []string{},
+	Skip:         false,
+	SetupConfig:  func(config *config.AppConfig) {},
+	SetupRepo: func(shell *Shell) {
+		shell.NewBranch("feature/TEST-002")
+		shell.CreateFile("test-wip-commit-prefix", "This is foo bar")
+	},
+	Run: func(t *TestDriver, keys config.KeybindingConfig) {
+		t.Views().Commits().
+			IsEmpty()
+
+		t.Views().Files().
+			IsFocused().
+			PressPrimaryAction().
+			Press(keys.Files.CommitChangesWithoutHook)
+
+		t.ExpectPopup().CommitMessagePanel().
+			Title(Equals("Commit summary")).
+			Type("foo bar").
+			Confirm()
+
+		t.Views().Commits().Focus()
+		t.Views().Main().Content(Contains("foo bar"))
+		t.Views().Extras().Content(Contains("--no-verify"))
```

Comment:
This is an interesting way of checking whether we passed the no-verify flag, but I'm not sure I like it much. If you had a test that committed twice, once with and once without the flag, you couldn't use this check for the second commit.

I'd prefer it if we installed an actual hook (using the technique in `fail_hooks_then_commit_no_hooks.go`), and have it write something to a file; we can then check whether the file exists (or what it contains) using `t.FileSystem().PathPresent()` or `t.FileSystem().FileContent()`.